src/main/webapp/app/entities/programming-exercise/programming-exercise.state.js

```diff
@@ -13,7 +13,7 @@
             parent: 'entity',
             url: '/programming-exercise',
             data: {
-                authorities: ['ROLE_USER'],
+                authorities: ['ROLE_ADMIN', 'ROLE_TA'],
                 pageTitle: 'artemisApp.programmingExercise.home.title'
```

Comment:
why did the authority change here?

Reply:
the `ROLE_USER` was automatically generated by jHipster when we created the programming-exercise entity. I looked at `src/main/webapp/app/entities/exercise/exercise.state.js` and there the authorities are `ROLE_ADMIN` and `ROLE_TA`, so I copied that over.